WindowsServerDocs/identity/ad-ds/plan/security-best-practices/Appendix-D--Securing-Built-In-Administrator-Accounts-in-Active-Directory.md

```diff
@@ -61,11 +61,11 @@ For the built-in Administrator account in each domain in your forest, you should
         -   Deny log on through Remote Desktop Services
 
 > [!NOTE]
-> These settings will ensure that the domain's built-in Administrator account cannot be used to connect to a domain controller, although the account, if enabled, can log on locally to domain controllers. Because this account should only be enabled and used in disaster-recovery scenarios, it is anticipated that physical access to at least one domain controller will be available, or that other accounts with permissions to access domain controllers remotely can be used.
+> These settings will ensure that the domain's built-in Administrator account cannot be used to connect to a domain controller, although the account can log on locally to domain controllers. Because this account should only be used in disaster-recovery scenarios, it is anticipated that physical access to at least one domain controller will be available, or that other accounts with permissions to access domain controllers remotely can be used.
 
 -   Configure Auditing of Administrator Accounts
 
-    When you have secured each domain's Administrator account and disabled it, you should configure auditing to monitor for changes to the account. If the account is enabled, its password is reset, or any other modifications are made to the account, alerts should be sent to the users or teams responsible for administration of Active Directory, in addition to incident response teams in your organization.
+    When you have secured each domain's Administrator account, you should configure auditing to monitor for usage of, or changes to, the account. If the account is used to logon, its password is reset, or any other modifications are made to the account, alerts should be sent to the users or teams responsible for administration of Active Directory, in addition to incident response teams in your organization.

```

Comment:
'to log on' is a compound verb, while 'logon' is the noun. Anyway, I think it should be phrased more like this:
```suggestion
    When you have secured each domain's Administrator account, you should configure auditing to monitor for usage of, or changes to, the account. If the account is logged in to, its password is reset, or any other modifications are made to the account, alerts should be sent to the users or teams responsible for administration of Active Directory, in addition to incident response teams in your organization.
```
Feel free to suggest a different phrasing, as long as it fits the general Microsoft Docs conventions.